docs/admin/metrics/serving-metrics-api.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Knative Serving metrics API"
+weight: 10
+type: "docs"
+aliases:
+  - /docs/serving/metrics/#admin
+---
+
+Administrators can monitor the Knative Serving control plane by looking at the metrics exposed by each Knative Serving component.
```

Comment:
ditto. data plane.

Reply:
well :) reading through these, maybe both control and dataplane? Which makes me think if they should be grouped then differently? Thinks like Controller / Webhook vs. Activator. I think some of these are more applicable for the Application developer vs. cluster admin, but just jotting it down and get somebody from serving to help :)

Reply:
Yes, always jot it down, this is all good context 😄

Is there a description of control vs dataplane anywhere that we can reuse, or can we create a blurb about each of those to explain the difference in the metrics topics? @skonto is explaining the difference between these something you could help with?

Reply:
Updated these to say data plane and control plane, can you open a follow up issue to improve these docs maybe please and loop in SMEs so we're not blocking the PR on this?